js-exercises/alternating-characters/alternatingCharacters.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-plusplus */
+const alternatingCharacters = (arrOfInputStrings = []) =>
+  // eslint-disable-next-line implicit-arrow-linebreak
+  arrOfInputStrings.map(input => {
+    let minimumDeletionCount = 0;
+    const everyInputArr = input.split('');
```

Comment:
`everyInputArr` does not sound like a good variable name. It is a little confusing

Reply:
yep, inputArr sounds better now :)